wordpress-mobile/WordPress-Android
# Gutenberg image fullscreen preview

Fixes  [gb-mobile 1286](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1286)

### Related PR's:
`gutenberg-mobile` changes: https://github.com/wordpress-mobile/gutenberg-mobile/commit/a20bcd6ddd9e25c74e1eca5729f17ae4fa76057f
`gutenberg` changes: https://github.com/cameronvoell/gutenberg/commit/0a10c343934c9d6753fbcb50d21bdc808d00ee01

### To test:

1. Open article in the block editor.
2. Navigate to an image block with an image loaded, or add a new image block and add an image
3. Select the image block and notice the blue border shows around the image
4. Press the image while the image block is selected and notice the image will be viewed in fullscreen.

### PR submission checklist:
